### DISCUSSION TITLE
Lidarr Integration

### DISCUSSION THREAD

**Discussion by MonsterBandit** [sct/overseerr#2564] [Feature Requests]
Added this to my dock-arr setup recently and love it.  Was wondering if there were any plans to add Lidarr integration in the future?

**Comment by TheCatLady** [comment]
No plans at this time.

**Reply by KINGLIFER** [reply]
Please consider. For some reason no one love lidarr... lol. But the competition does have this implemented but in no way shape or fashion looks as good as OVrSr.

**Comment by paloindici** [comment]
+1 vote for lidarr integration, it's the only thing missing from Overseerr !👍

**Reply by triksmelb** [reply]
+1 for lidarr / music intergration - would make overseerr a one stop shop for media.

**Comment by Drallas** [comment]
+1 lidarr / music integration, it would make us so happy; also less manual work for me.

**Comment by enigmaoftech** [comment]
+1 for Lidarr integration

**Comment by stolirocks1** [comment]
Lidarr integration is the only reason I'm still using ombi just for that. And I hate ombi

**Comment by TubaYogy** [comment]
Big +1 for Lidarr as Well as it will make and excellent product better

**Comment by TheHarbynger** [comment]
+1 for Lidarr support. Don't really want to setup Ombi, but am considering it for Lidarr support so I don't have to keep managing music for my wife and kids.

**Comment by oOStroudyOo** [comment]
+1

**Comment by Ninjadude101** [comment]
+1

**Comment by binarygeek119** [comment]
Musicbrainz may be a good metadata source. If that's the issue.

**Comment by ano0002** [comment]
I'm trying to implement it here, I'll try to keep you guys updated [Github Branch](https://github.com/ano0002/overseerr/tree/overseer-with-lidarr-support). I've already implemented most of the musicbrainz api, I now need to integrate Lidarr one and then I'll be able to start the front-end integration

**Reply by ano0002** [reply]
Btw, I've just started on this but I'll still like to not be spammed on when it'll be ready. I'll go at my pace.

**Reply by MonsterBandit** [reply]
Awesome.  Thank you so much.  I shall continue to wait patiently.

**Comment by daath** [comment]
+1

**Comment by cociweb** [comment]
+1

**Comment by byakuya322** [comment]
I also vote for this
